File: TrueFalseQuiz/app/src/main/java/com/comp1601/truefalsequiz/Question.java
package com.comp1601.truefalsequiz;

public class Question {
    private String question;
    private String answer;

    /**
     *
     * @param questionString
     */
    public Question(String questionString){
        //questionString is expected to be of the form:
        //"Is Java an Object-Oriented Language?[Yes]"
        String[] splitQuestion = questionString.split("\\[");
        splitQuestion[1] = splitQuestion[1].replace("]", "");

        question = splitQuestion[0];
        answer =splitQuestion[1];

    }

    public String getQuestion(){
        return answer;
    }

    public String getAnswer(){
        return question;
    }
}

File: Multiple_Choice_Quiz_App/app/src/main/java/com/comp1601/multiple_choice_quiz_app/Question.kt
package com.comp1601.multiple_choice_quiz_app



/**
 * @param questionString: a string containing the question followed by its answer in square brackets
 * ex: "Is Java an Object-Oriented Language?[E]"
 */
class Question (questionString: String){

    //Define class properties
    private var question: String
    private var answer: String


    init {
        //class initializer code:
        val splitQuestion: MutableList<String> = questionString.split("\\[").toMutableList()
        splitQuestion[1] = splitQuestion[1].replace("]", "")
        this.question = splitQuestion[0]
        this.answer = splitQuestion[1]
    }


    public fun getQuestion():String {return question}
    public fun getAnswer():String {return answer}


}